runtime/crab/src/lib.rs

```diff
@@ -792,6 +792,16 @@ impl darwinia_relayer_game::Trait<EthereumRelayerGameInstance> for Runtime {
 
 impl darwinia_header_mmr::Trait for Runtime {}
 
+parameter_types! {
+	pub const CrabIssuingModuleId: ModuleId = ModuleId(*b"da/crabk");
```

Comment:
"da/crais" is better than "da/crabk"

"da/crabk" could be used later on Darwinia side.